src/sentry/static/sentry/app/components/dataExport.tsx

```diff
@@ -126,7 +126,7 @@ class DataExport extends React.Component<Props, State> {
 const NewButton = ({children, ...buttonProps}) => (
   <Button {...buttonProps}>
     {children}
-    <FeatureBadge type="new" />
+    <FeatureBadge type="new" noTooltip />
```

Comment:
Should this just be default for all FeatureBadges?

Reply:
I think so, but I've not looked into what impacts that would have elsewhere.